packages/now-cli/src/util/dev/server.ts

```diff
@@ -1547,6 +1605,55 @@ export default class DevServer {
     }
     return false;
   }
+
+  async runDevCommand() {
+    if (!this.devCommand) return;
+
+    const cwd = this.cwd;
+
+    const { stdout: yarnBinStdout } = await execAsync('yarn', ['bin'], {
+      cwd,
+    });
+
+    const nodeBinPath = yarnBinStdout.trim();
```

Comment:
```suggestion
    const yarnBinPath = yarnBinStdout.trim();
```

Reply:
It's still the `node_modules` bin folder 🤔 @TooTallNate

Reply:
I don't really mind though. It looks good too, I'll update 👍